Resources/views/Collector/db.html.twig

```diff
@@ -447,9 +447,17 @@
             var targetElement = document.getElementById(targetId);
 
             if (targetElement.style.display != 'block') {
-                Sfjs.load(targetId, link.href, null, function(xhr, el) {
-                    el.innerHTML = 'An error occurred while loading the query explanation.';
-                });
+                if (targetElement.getAttribute('data-sfurl') !== link.href) {
+                    fetch(link.href, {
+                        headers: {'X-Requested-With': 'XMLHttpRequest'}
```

Comment:
what's this header needed for?

Reply:
Our code relies on `$request->isXmlHttpRequest()` to decide whether the rendered content should extend the ajax layout of the profiler (which only contains a `{% block panel %}` to avoid rendering the full page or no) or the full layout. This method is based on this header to detect that (which is a de-facto standard shared by many JS frameworks in the past, like jQuery, Prototype and Mootools to allow telling the backend that this is an ajax request)